service/scaladsl/client/src/main/scala/com/lightbend/lagom/scaladsl/client/ServiceClient.scala

```diff
@@ -112,10 +113,13 @@ trait LagomServiceClientComponents extends TopicFactoryProvider {
   def serviceInfo: ServiceInfo
   def serviceLocator: ServiceLocator
   def materializer: Materializer
+  def actorSystem: ActorSystem
   def executionContext: ExecutionContext
   def environment: Environment
   def applicationLifecycle: ApplicationLifecycle
 
+  lazy val circuitBreakerMetricsProvider: CircuitBreakerMetricsProvider = new CircuitBreakerMetricsProviderImpl(actorSystem)
+
```

Comment:
`LagomServiceClientComponents` will always have a default `CircuitBreakerMetricsProvider`. 
A client application may only use topic client (and not really need `CircuitBreakerMetricsProvider`)  but I thought that client applications usually will want this feature.

The exception are non lagom applications that use a `LagomClientFactory`. I'd have to review that `CircuitBreakerMetricsProvider` is not created when using a `LagomClientFactory`